pkg/hooks/launch.go

```diff
@@ -45,10 +45,10 @@ func (h *Hook) LaunchUserApplication(appCmd, appContainer, appNetwork string, De
 
 	if appCmd == "" {
 		if len(appContainer) == 0 {
-			return fmt.Errorf(Emoji + "please provide container name when running application container in isolation")
+			return fmt.Errorf("%s please provide container name when running application container in isolation", Emoji)
```

Comment:
incase adding the string like here replacement is not required. Like this log won't generate the error

Reply:
Yes, I understand that here, it won't generate the error, but I think even though `Emoji + "` is syntactically correct, it can lead to issues if the Emoji variable is empty or not a string. It also makes the code less readable.
That's why I made these changes also.

So should I make changes `fmt.Errorf(Emoji + "failed to get Resolver")` only?

Reply:
That's a great approach @jaimin001 but the emoji in this context is always a string that has a defined value. Thus keeping it to the original syntax is the better thing to do.

Reply:
Seems there had been some confusion @jaimin001. What I meant was to replace the ```fmt.Errorof(Emoji, ``` and not the addition of string.